src/states/node.rs

```diff
@@ -1302,18 +1210,6 @@ impl Node {
                     self.routing_msg_backlog.push(routing_msg);
                     return Ok(());
                 }
-                ConnectionInfoRequest { .. } => {
```

Comment:
Why did this go away, it does not seem to be a ProvingNode thing?

Reply:
OK, so the difference regarding this between the fleming branch and this PR is only in how we handle the case when we are already approved, but not yet established and we receive `ConnectionInfoRequest` whose `src.name` matches out `joining_prefix`. In fleming, we would handle the message, but after this PR we only backlog it. This may or might not be incorrect. I guess the question is whether non-established node needs to handle connection requests now, or whether it is OK for it to wait until it becomes established. In case we need to handle them, another question is what to use in place of `joining_prefix`, as `Node` no longer has that member. I guess `chain.our_prefix()` ?

Reply:
I do not know if it is correct or not, but if we are unsure, maybe better to keep it as it was?
Also, since this is a refactoring PR, we should probably explicitly document any change of behaviour in the PR description.

Reply:
You are probably right. I'll keep the original behaviour.